### DISCUSSION TITLE
Start window in browser doesnt initiate

### DISCUSSION THREAD

**Discussion by RCaleta1** [lllyasviel/Fooocus#3400] [Q&A] [answered]
Hi, I tried to installed, Fooocus and everything run s smooth with installation, but never initiate the starting window.
I have tried on 2 different computers, after extraction ends, press any key..... appears and after that command window  closes and nothing happens. Please help!!!!!I need it for realistic image creation. I tried on Huawai comp and on Dell comp.

**Comment by mashb1t** [comment]
> Huawai comp and on Dell comp

Do you have a dedicated GPU? You may try to use Fooocus after modifying the run.bat file and adding `--always-cpu` after entry_with_update.py. It will take significantly longer to generate an image with CPU only, but at least it's possible then.

Please check if there are any errors in the terminal log and post your full log from first line to last line (pause) for further debugging.

**Reply by mashb1t** [reply]
Appendix: make sure your PC meets the minimum system requirements, which you can find in the readme.

**Reply by RCaleta1** [reply]
Hi... 
D:\FOOKUS\Fooocus_win64_2-5-0\Fooocus_win64_2-5-0>.\python_embeded\python.exe -s Fooocus\entry_with_update.py --always-cpu--preset realistic
Already up-to-date
Update succeeded.
[System ARGV] ['Fooocus\\entry_with_update.py', '--always-cpu--preset', 'realistic']
Python 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)]
Fooocus version: 2.5.2
usage: entry_with_update.py [-h] [--listen [IP]] [--port PORT] [--disable-header-check [ORIGIN]]
                            [--web-upload-size WEB_UPLOAD_SIZE] [--hf-mirror HF_MIRROR]
                            [--external-working-path PATH [PATH ...]] [--output-path OUTPUT_PATH]
                            [--temp-path TEMP_PATH] [--cache-path CACHE_PATH] [--in-browser] [--disable-in-browser]
                            [--gpu-device-id DEVICE_ID] [--async-cuda-allocation | --disable-async-cuda-allocation]
                            [--disable-attention-upcast] [--all-in-fp32 | --all-in-fp16]
                            [--unet-in-bf16 | --unet-in-fp16 | --unet-in-fp8-e4m3fn | --unet-in-fp8-e5m2]
                            [--vae-in-fp16 | --vae-in-fp32 | --vae-in-bf16] [--vae-in-cpu]
                            [--clip-in-fp8-e4m3fn | --clip-in-fp8-e5m2 | --clip-in-fp16 | --clip-in-fp32]
                            [--directml [DIRECTML_DEVICE]] [--disable-ipex-hijack]
                            [--preview-option [none,auto,fast,taesd]]
                            [--attention-split | --attention-quad | --attention-pytorch] [--disable-xformers]
                            [--always-gpu | --always-high-vram | --always-normal-vram | --always-low-vram | --always-no-vram | --always-cpu [CPU_NUM_THREADS]]
                            [--always-offload-from-vram] [--pytorch-deterministic] [--disable-server-log]
                            [--debug-mode] [--is-windows-embedded-python] [--disable-server-info] [--multi-user]
                            [--share] [--preset PRESET] [--disable-preset-selection] [--language LANGUAGE]
                            [--disable-offload-from-vram] [--theme THEME] [--disable-image-log] [--disable-analytics]
                            [--disable-metadata] [--disable-preset-download] [--disable-enhance-output-sorting]
                            [--enable-auto-describe-image] [--always-download-new-model]
                            [--rebuild-hash-cache [CPU_NUM_THREADS]]
entry_with_update.py: error: unrecognized arguments: --always-cpu--preset realistic

D:\FOOKUS\Fooocus_win64_2-5-0\Fooocus_win64_2-5-0>pause
Press any key to continue . . . this is the message I get.

I see you are in my area, is it possible to meet with you....or to have your real email.

I installed Python, and I have installed the FFPMG, and Miniconda, but somehow...it is not connected...

**Reply by mashb1t** [reply]
add a space between the arguments
`.\python_embeded\python.exe -s Fooocus\entry_with_update.py --always-cpu --preset realistic`

**Reply by RCaleta1** [reply]
Where to put this code in run windows batch?

**Reply by RCaleta1** [reply]
now I get this
D:\FOOKUS\Fooocus_win64_2-5-0\Fooocus_win64_2-5-0>.\python_embeded\python.exe -s entry_with_update.py --always-cpu --preset realistic
.\python_embeded\python.exe: can't open file 'D:\\FOOKUS\\Fooocus_win64_2-5-0\\Fooocus_win64_2-5-0\\entry_with_update.py': [Errno 2] No such file or directory

D:\FOOKUS\Fooocus_win64_2-5-0\Fooocus_win64_2-5-0>pause
Press any key to continue . . .

**Reply by RCaleta1** [reply]
please do you have an email so that I can directly  communicate with you?

**Reply by mashb1t** [reply, marked as answer]
You can execute these 2 commands or modify and execute the run.bat.

```
cd D:\FOOKUS\Fooocus_win64_2-5-0\Fooocus_win64_2-5-0\Fooocus
..\python_embeded\python.exe -s entry_with_update.py --always-cpu --preset realistic
```

**Reply by RCaleta1** [reply]
It is loading the start page now, I copied this and it started loading.Please I tried to contact you on Linked in, please I am in the same area , I live in Freudenstadt BW, please contact me directly on email, I need your help with installation, my email is chaleta@gmail.com. Please

**Reply by RCaleta1** [reply]
D:\FOOKUS\Fooocus_win64_2-5-0\Fooocus_win64_2-5-0\Fooocus>..\python_embeded\python.exe -s entry_with_update.py --always-cpu --preset realistic
Already up-to-date
Update succeeded.
[System ARGV] ['entry_with_update.py', '--always-cpu', '--preset', 'realistic']
Python 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)]
Fooocus version: 2.5.2
Loaded preset: D:\FOOKUS\Fooocus_win64_2-5-0\Fooocus_win64_2-5-0\Fooocus\presets\realistic.json
[Cleanup] Attempting to delete content of temp dir C:\Users\USER~1\AppData\Local\Temp\fooocus
[Cleanup] Cleanup successful
Running on 14 CPU threads
Total VRAM 16111 MB, total RAM 16111 MB
Set vram state to: DISABLED
Always offload VRAM
Device: cpu
VAE dtype: torch.float32
Using sub quadratic optimization for cross attention, if you have memory or speed issues try using: --attention-split
Refiner unloaded.
Running on local URL:  http://127.0.0.1:7865

To create a public link, set `share=True` in `launch()`.
IMPORTANT: You are using gradio version 3.41.2, however version 4.29.0 is available, please upgrade.
--------

D:\FOOKUS\Fooocus_win64_2-5-0\Fooocus_win64_2-5-0\Fooocus>

**Reply by mashb1t** [reply]
As of the readme and your console logs there are no more steps left to install Fooocus, but [minimum system specs](https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#minimal-requirement) for CPU only are 32GB RAM, which your PC doesn't have (16GB).
You may use sufficient swap (40GB) but it may still not work as you're not meeting minimum required specs.

**Reply by RCaleta1** [reply]
Thank you very much!Although I would like to make a contact with you and if it is possible to even make some training for me.
You have my mail...thank you once more!
